tests/ui/macros/expr_2021_old_edition.stderr

```diff
@@ -0,0 +1,26 @@
+error: invalid fragment specifier `expr_2021`
+  --> $DIR/expr_2021_old_edition.rs:6:6
+   |
+LL |     ($e:expr_2021) => {
+   |      ^^^^^^^^^^^^
+   |
+   = help: fragment specifier `expr_2021` requires Rust 2021 or later
+           valid fragment specifiers are `ident`, `block`, `stmt`, `expr`, `pat`, `ty`, `lifetime`, `literal`, `path`, `meta`, `tt`, `item` and `vis`
```

Comment:
I would probably split this into two separate `help` messages but er, doesn't matter

Reply:
Yeah, I think that'd be better. Let me poke around at it some more, I'm still getting the hang of the diagnostics system.